__tests__/controllers/users/update-user.spec.ts

```diff
@@ -6,7 +6,7 @@ import { User } from '@entities/user.entity';
 import { API } from '../../utils';
```

Comment:
Make sure that we are pointing to the dev DB and not production. WE dont want new fake users (?

Reply:
Made sure it is pointing to the dev db.